iina/FileGroup.swift

```diff
@@ -144,8 +143,7 @@ class FileGroup {
 
   func flatten() -> [String: [FileInfo]] {
     var result: [String: [FileInfo]] = [:]
-    var search: ((FileGroup) -> Void)!
-    search = { group in
+    func search(_ group: FileGroup) {
```

Comment:
At first I thought maybe `[weak search]`? But Swift complains about "'weak' may only be applied to class and class-bound protocol types, not '(FileGroup) -> Void'".